packages/core/src/internal/proc.js

```diff
@@ -263,9 +263,6 @@ export default function proc(env, iterator, parentContext, parentEffectId, meta,
         mainTask.cont(result.value)
       }
     } catch (error) {
-      if (mainTask._isCancelled) {
-        env.logError(error)
```

Comment:
This should not be removed, or else the error will be swallowed...
If you log `mainTask.cont` in this if-statement, it will print as `noop`, so the error object will be sent to `noop`.  
JYI, `mainTask.cont` is set to `noop` by `forkQueue` when we cancel the task.

Reply:
yes, I was going to throw here, but forgot as tests 'passed'